docs/src/pages/[platform]/components/text/examples/TextThemeExample.tsx

```diff
@@ -0,0 +1,28 @@
+import { Text, Flex, ThemeProvider, Theme } from '@aws-amplify/ui-react';
+
+const theme: Theme = {
+  name: 'text-theme',
+  tokens: {
+    components: {
+      text: {
+        color: { value: '{colors.green.40}' },
+        primary: {
+          color: { value: '{colors.teal.40}' },
+        },
+        warning: {
+          color: { value: '{colors.pink.60}' },
+        },
+      },
+    },
+  },
+};
+
+export const TextThemeExample = () => (
+  <ThemeProvider theme={theme}>
+    <Flex>
+      <Text>Default</Text>
```

Comment:
Looks like `default` and `primary` color don't meet the contrast ratio in light mode. I feel like it is so hard to pick a right but also nice color in both light/dark mode.
![Screen Shot 2022-06-15 at 3 15 49 PM](https://user-images.githubusercontent.com/40295569/173941559-51de967a-b968-46b2-86f2-fe94ae83c11d.png)

![Screen Shot 2022-06-15 at 3 15 57 PM](https://user-images.githubusercontent.com/40295569/173941630-671031b7-0b34-499a-a19d-6c9aa51ae92d.png)